File: 진우/14weeks/11050. 이항 계수 1/이항 계수 1.py
# N, K 입력
N, K = map(int, input().split())

# N! 계산
n = 1
for i in range(N, 0, -1):
    n *= i

# K! 계산
k = 1
for i in range(K, 0, -1):
    k *= i

# (N-K)! 계산
r = 1
for i in range(N - K, 0, -1):
    r *= i

# 이항계수 nCk = N! / (K! * (N-K)!)
result = n // (k * r)

# 결과 출력
print(result)

File: 진우/14weeks/11050. 이항 계수 1/이항 계수 1.cc
#include <iostream>

using namespace std;

int main() {
    int N, K;
    cin >> N >> K;      // N과 K 입력

    long long n = 1;
    // N! 계산
    for (int i = N; i >= 1; i--) {
        n *= i;
    }

    long long k = 1;
    // K! 계산
    for (int i = K; i >= 1; i--) {
        k *= i;
    }

    long long r = 1;
    // (N-K)! 계산
    for (int i = (N - K); i >= 1; i--) {
        r *= i;
    }

    // 이항계수 nCk = N! / (K! * (N-K)!)
    int result = n / (k * r);

    cout << result << "\n";  // 결과 출력

    return 0;
}
